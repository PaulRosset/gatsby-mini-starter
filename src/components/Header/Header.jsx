import React from 'react';
import styled from 'styled-components';

export const ContainerHeader = styled.div`
  background: #0e6eb8;
  margin-bottom: 1.45rem;
`;

export const FormHeader = styled.div`
  margin: 0 auto;
  max-width: 960px;
  padding: 1.45rem 1.0875rem;
`;

export const StyledLinkHeader = styled(Link)`
  color: white;
  text-decoration: none;

  &:hover {
    color: white;
  }
`;

const Header = () => (
  <ContainerHeader>
    <FormHeader>
      <h1>
        <StyledLinkHeader to="/">Gatsby</StyledLinkHeader>
      </h1>
    </FormHeader>
  </ContainerHeader>
);

export default Header;
