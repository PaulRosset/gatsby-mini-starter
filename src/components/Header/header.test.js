import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from './../../enzyme';

import Header from './Header';

describe('Render app without crash', () => {
  it('Rendering <Header />', () => {
    const header = shallow(<Header />);
    expect(toJson(header).toMatchSnapshot());
  });
});
