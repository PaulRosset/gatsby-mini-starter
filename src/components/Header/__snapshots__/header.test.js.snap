// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render app without crash Rendering <Header /> 1`] = `
<styled.div>
  <styled.div>
    <h1>
      <Styled(GatsbyLink)
        to="/"
      >
        Gatsby
      </Styled(GatsbyLink)>
    </h1>
  </styled.div>
</styled.div>
`;
